lib/taskana-core/src/main/java/pro/taskana/impl/persistence/MapTypeHandler.java

```diff
@@ -54,15 +55,16 @@ public Map getNullableResult(ResultSet rs, int columnIndex) throws SQLException
   }
 
   @Override
-  public Map getNullableResult(CallableStatement cs, int columnIndex) throws SQLException {
+  public Map<String, Object> getNullableResult(CallableStatement cs, int columnIndex)
```

Comment:
Can we use optional here? 
I always aim to minimize nulls ;)

Reply:
unfortunately we can't. We are extending the Mybatis `TypeHandler` class and are bound to its signature